android-core/src/androidTest/kotlin/com.mparticle/networking/MParticleBaseClientImplTest.kt

```diff
@@ -56,17 +214,112 @@ class MParticleBaseClientImplTest : BaseCleanInstallEachTest() {
         val baseClientImpl = AccessUtils.getApiClient() as MParticleBaseClientImpl
         for (endpoint in MParticleBaseClientImpl.Endpoint.values()) {
             val generatedUrl = baseClientImpl.getUrl(endpoint, endpoint.name, true)
-            Assert.assertEquals(defaultUrls[endpoint].toString(), generatedUrl.toString())
-            TestCase.assertTrue(generatedUrl === generatedUrl.defaultUrl)
+            assertEquals(defaultUrls[endpoint].toString(), generatedUrl.defaultUrl.toString())
         }
         for (endpoint in MParticleBaseClientImpl.Endpoint.values()) {
             val generatedUrl = baseClientImpl.getUrl(endpoint, endpoint.name, false)
             Assert.assertNotEquals(defaultUrls[endpoint].toString(), generatedUrl.toString())
             Assert.assertFalse(generatedUrl === generatedUrl.defaultUrl)
-            Assert.assertEquals(
+            assertEquals(
```

Comment:
Is there a reason that some checks use `Assert.assert...` and others just use `assert...`? In this case the existing `Assert.assertEquals` was even changed to remove the `Assert.` but the other asserts still have it?

Reply:
No, there’s no difference. When I rewrote this, the Assert class was automatically imported, so we can use assertEquals directly.